src/redis.c

```diff
@@ -196,6 +200,10 @@ static int redis_config_node(oconfig_item_t *ci) /* {{{ */
       status = cf_util_get_int(option, &timeout);
       if (status == 0)
         rn.timeout.tv_usec = timeout;
+    } else if (strcasecmp("Db", option->key) == 0) {
+      status = cf_util_get_int(option, &db);
+      if (status == 0)
+        rn.db = db;
```

Comment:
I think we don't need to use intermediate variable here and may store value to `rn->db` directly.
See other cases of `cf_util_get_int` in other plugins - most of them store values directly.
Also you can unify this way and fix `Timeout` option too.